script.rb

```diff
@@ -1,11 +1,16 @@
+# !/usr/bin/env ruby
 # Purpose : Write a script where asks for a confirmation when deleting a folder if this folder contains a file .important
 # If the user confirms the deletion, the folder is deleted.
 # If the user cancels the deletion, the folder is not deleted.
 # Author: Guillaume MORET
 # ------------------------------------------------------------------------------
 
-# require class folder.class.rb
-require_relative 'folder.class.rb'
+# require class folder.rb
+require 'byebug'
+require_relative 'folder.rb'
+
+
+Folder.delete_folder("../file_test2")
```

Comment:
```suggestion
Folder.delete_folder(ARGV[1])
```

(For example)

Prefer to use argument to allow choosing the target directory rather than `../file_test2`